server/src/main/java/org/opensearch/cluster/coordination/FollowersChecker.java

```diff
@@ -423,7 +426,11 @@ public String executor() {
         }
 
         void failNode(String reason) {
-            clusterManagerMetrics.incrementCounter(clusterManagerMetrics.followerChecksFailureCounter, 1.0);
+            clusterManagerMetrics.incrementCounter(
```

Comment:
Lets have both the metrics as this will then be a breaking change for downstream